python/ray/autoscaler/aws/example-full.yaml

```diff
@@ -69,7 +69,7 @@ head_node:
     BlockDeviceMappings:
         - DeviceName: /dev/sda1
           Ebs:
-              VolumeSize: 50
+              VolumeSize: 100
```

Comment:
Do you know where all the size goes? Even though the disk space is very cheap, 100gb image sounds unnecessarily heavy for a simple example.

Reply:
No clue but the DLAMI can't run otherwise......